client/components/main/header/Header.jsx

```diff
@@ -23,6 +23,12 @@ const Header = () => {
     setActiveBurger(!activeBurger);
   };
 
+  const handleKeyDown = e => {
+    if (e.key !== 'Tab' && e.key !== 'Shift' && e.key !== 'Alt') {
```

Comment:
Same comment on keypresses.

Reply:
Nice point Adam, made the changes to 'Space' and 'Enter'